client/clientservice/include/client/clientservice/state_snapshot_service.hpp

```diff
@@ -37,6 +37,11 @@ class StateSnapshotServiceImpl final
                         vmware::concord::client::statesnapshot::v1::ReadAsOfResponse* response) override;
 
  private:
+  bool isHashValid(uint64_t snapshot_id,
```

Comment:
Can we make this method `const`?

Reply:
Yes, I will add this in the next commit.